diff --git a/node_modules/ssb-db2/indexes/plugin.js b/node_modules/ssb-db2/indexes/plugin.js
index bbe9b29..72bf26b 100644
--- a/node_modules/ssb-db2/indexes/plugin.js
+++ b/node_modules/ssb-db2/indexes/plugin.js
@@ -11,7 +11,7 @@ const Debug = require('debug')
 const DeferredPromise = require('p-defer')
 const { indexesPath } = require('../defaults')
 
-const notInABrowser = typeof window === 'undefined'
+const notInABrowser = true
 let rimraf
 let mkdirp
 if (notInABrowser) {